Makefile

```diff
@@ -1,5 +1,5 @@
 
-.PHONY: db test pep8 clean doc check-venv check-reqs check-venv
+.PHONY: db test pep8 clean doc check-venv check-venv
```

Comment:
You left check-venv here, but removed it in others.  No opinion, just an inconsistency (from what I've reviewed so far).